QUEST_LV_0200.tsv

```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
 QUEST_LV_0200_20150323_005237	You've defeated many demons. Talk to the disciple, Prosit.
-QUEST_LV_0200_20150323_005238	Talk to Gitise
-QUEST_LV_0200_20150323_005239	You can see Gitise who is worried a lot at the settlement land of Gitese. Talk with Gitese.
-QUEST_LV_0200_20150323_005240	Ask for help from Pyromancer Master at Klaipeda
-QUEST_LV_0200_20150323_005241	Gitese told you that the workers at the settlement land were attacked by the monsters and they are sick. Talk to Pyromance Master to get his help.
-QUEST_LV_0200_20150323_005242	Pyromancer Master told you that he will definitely help you. Talk to Pyromancer Master again.
+QUEST_LV_0200_20150323_005238	$Talk to Gitis
+QUEST_LV_0200_20150323_005239	$You can see Gitis who is worried a lot at the settlement land of Gitis. Talk with Gitis.
+QUEST_LV_0200_20150323_005240	$Ask for help from the Pyromancer Master at Klaipeda
+QUEST_LV_0200_20150323_005241	$Gitis told you that the workers at the settlement land were attacked by the monsters and they are sick. Talk to the Pyromancer Master to get her help.
+QUEST_LV_0200_20150323_005242	$The Pyromancer Master told you that he will definitely help you. Talk to the Pyromancer Master again.
 QUEST_LV_0200_20150323_005243	Collect the samples of pink Chuparukas at the settlement land
 QUEST_LV_0200_20150323_005244	{memo X}Pyromancer Master told you that he will definitely help you. First get some samples of pink Chuparukas at the settlement land.
-QUEST_LV_0200_20150323_005245	Hand it over to Pyromancer Master
-QUEST_LV_0200_20150323_005246	You've obtained the samples of pink Chuparukas. Hand them over to Pyromancer Master.
+QUEST_LV_0200_20150323_005245	$Hand it over to the Pyromancer Master
+QUEST_LV_0200_20150323_005246	$You've obtained the samples of pink Chuparukas. Hand them over to the Pyromancer Master.
 QUEST_LV_0200_20150323_005247	Obtain %s from the settlement land
-QUEST_LV_0200_20150323_005248	Pyromancer Master who was investigating the samples of the monsters doesn't feel good about something. Talk to Pyromancer Master.
+QUEST_LV_0200_20150323_005248	$The Pyromancer Master who was investigating the samples of the monsters doesn't feel good about something. Talk to the Pyromancer Master.
 QUEST_LV_0200_20150323_005249	Search for the clues of the massive chaos at the settlement land
-QUEST_LV_0200_20150323_005250	Pyromancer Master can feel some unknown poison and the spell. Go back to the settlement land and find out the true nature of the poison and the spell.
-QUEST_LV_0200_20150323_005251	You've found a leather and a dagger with the magic field drawn in them at Nuodo Bank. Take them to Pyromancer Master.
+QUEST_LV_0200_20150323_005250	$The Pyromancer Master can feel some unknown poison and the spell. Go back to the settlement land and find out the true nature of the poison and the spell.
```

Comment:
The Pyromancer Master can feel some unknown poison and the spell -> The Pyromancer Master can feel some unknown poison and magic?